src/Books.css

```diff
@@ -0,0 +1,272 @@
+.Books {
```

Comment:
- [ ] Please, consider fixing some points in styling, to be ready to get approval:

1. Here should be a category, which is come from adding a form at the bottom of a page, where you have to choose from the Category dropdown list.
2. Please try to add the divider, as it is also on original styling 👍 
3. Try to remove the bold effect to be closer to the original styling.
4. Try to remove the bold effect from the "Update progress" buttons 👍 

<img width="1284" alt="Screen Shot 2022-04-02 at 21 19 13 1" src="https://user-images.githubusercontent.com/49064106/161394301-86696bea-6a79-4cb6-bc04-498890d271aa.png">

5. Try to make the `hr` gray one. It will help to do that `border: solid 1px var(--grey-border);`
6. Please, consider adding the possibility to chose the category of a book here 👍 

<img width="1290" alt="Screen Shot 2022-04-02 at 21 23 47" src="https://user-images.githubusercontent.com/49064106/161394432-f4b5c620-b39e-4984-966d-ef49d4bff514.png">

Reply:
1. there is as there is no input for the category it remains as an empty string
2. sure
3. sure
4. sure
5. that would not work as hrs ignore border styling, sure can be done
6. that is beyond the styling and not part of this or previous steps